app/src/server/router/homes.ts

```diff
@@ -11,26 +11,44 @@ export const homesRouter = createProtectedRouter()
             if(!ctx.session.user){
                 return;
             }
-            const homes = await ctx.prisma.occupies.findMany({
+            const homes = await ctx.prisma.home.findMany({
                 select: {
-                    home: {
-                        select: {
-                            id: true,
-                            name: true,
-                            image: true,
-                            address: true,
-                        }
-                    }
+                    id: true,
+                    name: true,
+                    image: true,
+                    address: true,
                 },
                 where: {
-                    userId: ctx.session.user.id,
+                    Occupies: {
+                       some: {
+                            userId: ctx.session.user.id,
+                       }
+                    }
                 },
             });
+            
+            const homePromises = [];
             for(const home of homes){
-                if(home.home.image)
-                    home.home.image = await getSignedImage(home.home.image);
+                if(!home.image) continue;
```

Comment:
Thanks for fixing this. Might still be some other places where this occurs. Probably will fix @rhit-daviscl issue with image signing on testing for home crud? Not for sure if these are related